component/src/main/java/org/wso2/extension/siddhi/gpl/execution/streamingml/regression/adaptivemodelrules/util/AdaptiveModelRulesModel.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.extension.siddhi.gpl.execution.streamingml.regression.adaptivemodelrules.util;
+
+import com.yahoo.labs.samoa.instances.Instance;
+import moa.classifiers.core.driftdetection.ADWINChangeDetector;
+import moa.classifiers.core.driftdetection.ChangeDetector;
+import moa.classifiers.rules.AMRulesRegressor;
+import moa.classifiers.rules.core.anomalydetection.AnomalinessRatioScore;
+import moa.classifiers.rules.core.anomalydetection.AnomalyDetector;
+import moa.classifiers.rules.core.anomalydetection.NoAnomalyDetection;
+import moa.classifiers.rules.core.anomalydetection.OddsRatioScore;
+import moa.classifiers.rules.core.changedetection.NoChangeDetection;
+import moa.core.ObjectRepository;
+import moa.options.ClassOption;
+import moa.tasks.TaskMonitor;
+import org.apache.log4j.Logger;
+import org.wso2.extension.siddhi.gpl.execution.streamingml.regression.AbstractRegressor;
+import org.wso2.extension.siddhi.gpl.execution.streamingml.regression.Regressor;
+import org.wso2.extension.siddhi.gpl.execution.streamingml.util.MathUtil;
+import org.wso2.siddhi.query.api.exception.SiddhiAppValidationException;
+
+/**
+ * Represents AMRules Regressor model
+ */
+public class AdaptiveModelRulesModel extends AbstractRegressor implements Regressor {
+    private static final long serialVersionUID = 1L;
```

Comment:
Use uppercase for static values

Reply:
Declaring its own serialVersionUID should follow the following format. Field name should be the same.
 ANY-ACCESS-MODIFIER static final long serialVersionUID = 42L;

ref, [docs/Serializable](https://docs.oracle.com/javase/8/docs/api/java/io/Serializable.html)